target/linux/x86/image/grub-iso.cfg

```diff
@@ -3,7 +3,12 @@
 
 set default="0"
 set timeout="@TIMEOUT@"
-set root='(cd)'
+
+if [ x${grub_platform} = xefi ]; then
```

Comment:
I hate this notation, and it harkens back to a highly braindead Bourne shell from a couple of decades ago, which has thankfully disappeared.

Can we use:

```
if [ "$grub_platform" = "efi" ]; then
```

instead please?